aviary/variable_info/variable_meta_data.py

```diff
@@ -6481,23 +6566,45 @@
 add_meta_data(
     Dynamic.Mission.SHAFT_POWER,
     meta_data=_MetaData,
-    historical_name={"GASP": 'SHP',
+    historical_name={"GASP": None,
+                     "FLOPS": None,
+                     "LEAPS1": None
+                     },
+    units='hp',
+    desc='current shaft power, per engine'
+)
+
+add_meta_data(
+    Dynamic.Mission.SHAFT_POWER_MAX,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
                      "FLOPS": None,
                      "LEAPS1": None
                      },
     units='hp',
-    desc='The shaft horsepower'
+    desc='The maximum possible shaft power currently producible, per engine'
 )
 
 add_meta_data(
     Dynamic.Mission.SHAFT_POWER_CORRECTED,
     meta_data=_MetaData,
-    historical_name={"GASP": 'SHPCOR',
```

Comment:
was this historic name intentionally removed?

Reply:
I don't think that was intentional on my part, I'll add it back!